### PR TITLE
Add ProxiTok

### PR DESCRIPTION
**Pull request by dngray**
- [x] Please check this box to confirm you have disclosed any relevant conflicts of interest in your post.
- [x] Please check this box to confirm your agreement to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute your contribution as part of our project.
- [x] Please check this box to confirm you are the sole author of this work, or that any additional authors will also reply to this PR on GitHub confirming their agreement to these terms.

This came up in the discussion thread https://discuss.privacyguides.net/t/add-proxitok-to-frontends-in-software/11370

It could be useful, when wanting to view a TikTok through Tor.

### CODE REVIEW

**Comment by freddy-m** [comment]
> It could be useful

It if only _could_ be useful and we're unsure how stable it is then we shouldn't rush into adding it.

**Review by freddy-m — APPROVED** [review]
LGTM

File: docs/frontends.en.md [hunk 1]
```diff
@@ -61,6 +61,30 @@ When self-hosting, it is important that you have other people using your instanc
 
 When you are using a Nitter instance, make sure to read the privacy policy of that specific instance. Nitter instances can be modified by their owners and therefore may not reflect the default policy. Some instances have Tor .onion addresses which may grant some privacy as long as your search queries don't contain PII (Personally Identifiable Information).
 
+## Tik Tok
```
[the remark this answers is not in the document]
Comment (freddy-m) [reply docs/frontends.en.md:64]: Good spot!

File: docs/frontends.en.md [hunk 2]
```diff
@@ -61,6 +61,30 @@ When self-hosting, it is important that you have other people using your instanc
...
+    [:octicons-info-16:](https://github.com/pablouser1/ProxiTok/wiki){ .card-link title=Documentation}
+    [:octicons-code-16:](https://github.com/pablouser1/ProxiTok){ .card-link title="Source Code" }
+
+!!! tip
+
+    ProxiTok is useful if you want to disable JavaScript in your browser, such as [Tor Browser](https://www.torproject.org/) on the Safest security level.
+
+When self-hosting, it is important that you have other people using your instance as well in order for you to blend in. You should be careful with where and how you are hosting ProxiTok, as other peoples' usage will be linked to your hosting.
+
+When you are using an ProxiTok instance, make sure to read the privacy policy of that specific instance. ProxiTok instances can be modified by their owners and therefore may not reflect their associated privacy policy. Some instances have Tor .onion addresses which may grant some privacy as long as your search queries don't contain PII (Personally Identifiable Information).
```
Comment (mfwmyfacewhen) [inline docs/frontends.en.md:86]:
```suggestion
When you are using a ProxiTok instance, make sure to read the privacy policy of that specific instance. ProxiTok instances can be modified by their owners and therefore may not reflect their associated privacy policy. Some instances have Tor .onion addresses which may grant some privacy as long as your search queries don't contain PII.
```

**Review by mfwmyfacewhen — CHANGES_REQUESTED** [review]

**Comment by dngray** [comment]
> Is it stable/robust nowadays? I'd tried it a while ago to see if it could be considered, but it was inconsistent.

Well I was using it before, to [view](https://proxitok.pabloferreiro.es/%40polarbearsinternational/video/7168992335559363886) [some](https://proxitok.pabloferreiro.es/%40polarbearsinternational/video/7171515750316821802) [videos](https://proxitok.pabloferreiro.es/%40polarbearsinternational/video/7184475705768414506).

**Review by mfwmyfacewhen — APPROVED** [review]

**Comment by dngray** [comment]
bleh i should have made this a draft, as I was waiting on a SVG from the author, which I now have.

**Pull request merged**